MapboxSearch/sdk/src/main/java/com/mapbox/search/result/SearchResult.kt

```diff
@@ -181,6 +188,7 @@ public class SearchResult internal constructor(
     override fun hashCode(): Int {
```

Comment:
One more... perhaps we can find a way to reference it once

Reply:
Same